zarr/tests/test_storage_v3.py

```diff
@@ -88,6 +91,18 @@ def keys(self):
         """keys"""
 
 
+class DummyStorageTransfomer(StorageTransformer):
+    TEST_CONSTANT = "test1234"
+
+    extension_uri = "https://purl.org/zarr/spec/storage_transformers/dummy/1.0"
+    valid_types = ["dummy_type"]
+
+    def __init__(self, _type, test_value) -> None:
+        super().__init__(_type)
+        assert test_value == self.TEST_CONSTANT
```

Comment:
An assert within a helper class, even if only used for tests, is a little unconventional